### ISSUE TITLE
Data set partitioning

### ISSUE THREAD

**Issue by xiaog221** [LowLevelAI/GPP-LLIE#8]
Hello authors, I would like to ask if the test set used for reasoning in this paper is the same as the validation set used during training? Also, is the generation prior of the lolv2real dataset not provided?
